EDI/Operation/05_ManageDataSources.md

```diff
@@ -0,0 +1,156 @@
+Tip: If you want to map the fields directly after you have specified an EDIFACT field in the file settings, it is recommended to create the data source before specifying the header &frasl; line items file segments.
+
+
+
+## Assign data source
+
+Assign the data source from which you want to retrieve the data with which you want to fill the EDIFACT message fields.
+
+#### Prerequisites
+
+At least one definition setting has been created, see [Create basic definition settings](./01_ManageDefinitions.md#create-basic-definition-settings).
+>[INFO] The *Data sources* tab is visible only if you have created and saved a basic definition setting. 
+
+#### Procedure
+
+*DataHub Exporter > Exports*
+
+![Export definitions](../../Assets/Screenshots/EDI/Operation/ExportDefinitions.png "[Export definitions]")
+
+1. Click the exporter definition for which you want to define the data source.   
+    The *Edit definition "Definition name"* view is displayed. The *Settings* tab or the last opened tab is displayed by default.
+
+2. Click the *Data sources* tab.   
+    The *Data sources* tab is displayed. Because you have not yet created a data source, *No data source* is displayed in the list. 
+
+    ![Data sources tab](../../Assets/Screenshots/EDI/Operation/DataSourcesList.png "[Data sources tab]")
+
+3. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.     
+    The *Select source* pop-up window is displayed.
+
+    ![Select source](../../Assets/Screenshots/EDI/Operation/DataSourcesSelectSource.png "[Select source]")
+
+4. Select the data source in the *Entity type* drop-down list.   
+    - If desired, use the *RetailSuite Business document* entity type when you are currently defining the data source for a header file.    
+    You typically use this entity type for the header data when you want to transfer data with its origin from the *Actindo Core1 Platform*.
+    - If desired, use the *Business document position* entity type when you are currently defining the data source for a line item definition.   
+    You typically use this entity type for the line item data when you want to transfer line item data with its origin from the *Actindo Core1 Platform*.  
+    - If desired, use any other data source that provides you with data needed for the transfer.   
+    For example, you want to send an EDIFACT message with invoice data from SAP. In this case select the related SAP data source with which you can access the data needed for the header&frasl;line item file.   
+    
+    The *Attribute set* drop-down list is automatically filled with the data source defined in the *Entity type* field.
+    
+5. Click the [CONTINUE] button in the bottom right corner.   
+    The *Map attributes* pop-up window is displayed. If you have not yet defined string attributes in your file settings, the *There are no mappable fields in the destination* text is displayed. You can ignore this step for now.
```

Comment:
Vorschlag: *There are no mappable fields in the destination* message...

Reply:
done